templates/form_theme.html.twig

```diff
@@ -228,3 +228,20 @@
         <a href="{{ path('mooc_get_file', { slug: form.vars.value.slug, extension: form.vars.value.extension }) }}" target="_blank">Télécharger le fichier</a>
     {% endif %}
 {% endblock %}
+
+{% block _app_donation_confirmDonationType_row %}
+    {{ form_errors(form.parent.children.confirmSubscriptionAmount) }}
+    {%- for child in form %}
+        <p style="text-align: left">
```

Comment:
1) can you replace `p` with `div` pls? it's more correct
2) you can use `class="text--left"` to align the text instead of `style` attribute.
If I need to apply a style to my element, I am looking for it first in `/front/style`, maybe it already exists one class for this style.